packages/cmd/src/yargs/YargsConfigurer.ts

```diff
@@ -48,17 +48,8 @@ export class YargsConfigurer {
         /**
          * Add the command definitions to yargs
          */
-        const jsonResponseFormat =
```

Comment:
What was it about this code that was failing, causing you to use a different implementation in getResponseFormat( )?

Reply:
Previously we were checking `process.argv` when Imperative initialized to look for the `--response-format-json` argument. This worked in Zowe v1, but no longer works in Zowe v2 with daemon mode because:
* `process.argv` contains the arguments that the daemon server was launched with (`--daemon`), not the arguments passed to the current command.
* The CLI arguments need to be reloaded every time the daemon client issues a command, rather than being cached at the time Imperative initializes.